packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -174,6 +174,18 @@ multipleSubPointsContentInMobileWidth.play = (): void => {
         .setMarkdown(multipleSubPointsContent);
 };
 
+const differentListElementContentInSameLevel = `
+1. Point 1
+    * Sub point 1
+    1. Sub point 2`;
+
+export const differentListElementInSameLevel: StoryFn = createStory(mobileWidthComponent);
```

Comment:
I don't know if this visual test is worth having. Does it validate something unique that the unit test doesn't?

For example, if the content was:
```
1. Point 1
    * Sub point 1
    1. Sub point 2
    * Sub point 3
    1. Sub point 4
```
Then I could see that this would show whether sub poin 4 was numbered with a "1" or a "2". But as it is, it seems like this will not verify anything unique.

Reply:
I'll update the test have the list as suggested and it shows the fourth point to show as `1` considering it a new list after a different list element preceding it. But this case doesn't seem to be manually reproduceable.